charts/rancher-backup/files/default-resourceset-contents/rancher.yaml

```diff
@@ -36,6 +36,15 @@
   resourceNameRegexp: "management.cattle.io$|project.cattle.io$|catalog.cattle.io$|resources.cattle.io$"
 - apiVersion: "management.cattle.io/v3"
   kindsRegexp: "."
+  excludeKinds:
+    - "tokens"
```

Comment:
Won't this invalidate all kubeconfigs on the cluster ? And anything else using tokens ? What does the section below do ?

Reply:
Talked to Daishan: this part backs up everything _but_ tokens, the part below it backs up all tokens except those with provisioning label. So user kubeconfigs are fine.